### ISSUE TITLE
High-frequency distortion and potential bandwidth limitations in DiscreteVAE

### ISSUE THREAD

**Issue by zlh123123** [index-tts/index-tts#639]
### Description
I am currently fine-tuning the **IndexTTS1** model and have encountered significant audio artifacts, such as "crackling" and "electronic" noises, particularly when generating high-pitched audio segments.

After extensive testing over the past few weeks, I have ruled out the following as potential causes:
1. **Training Dataset:** The source data is clean and does not contain these artifacts.
2. **Vocoder:** My tests suggest the issue is not originating from the BigVGAN vocoder.

### Diagnostic Test & Observations
To isolate the bottleneck, I performed a frequency sweep test (0Hz to 24,000Hz):
- **Mel 1:** Original Mel-spectrogram generated from the 0-24kHz sweep signal.
- **Mel 2:** The reconstructed Mel-spectrogram after passing the signal through the **DiscreteVAE** (Linear -> Encoder -> Quantize -> Decoder).

**Comparison Result:**
By comparing **Mel 1** and **Mel 2**, I found that the reconstruction of high-frequency signals in **Mel 2** is very poor. There is a noticeable loss of detail and distortion in the upper-frequency range, which seems to be the root cause of the "robotic" sounds in high-pitched generations.

<img width="2100" height="1500" alt="Image" src="https://github.com/user-attachments/assets/a10ea989-6d28-4234-88a0-a46ab1ea70d6" />

### Questions
1. **Bandwidth Limitation:** Was the training data for the DiscreteVAE (Index V1) pre-processed with a low-pass filter (e.g., cutting off frequencies above 8,000Hz)?
2. **Training Strategy:** Are there specific constraints in the VAE architecture or the loss function (e.g., spectral loss weighting) that prioritize low-to-mid frequencies over high frequencies?
3. **Potential Fixes:** To resolve the high-frequency "crackling" in Index V1, would you recommend retraining the DiscreteVAE with a different configuration, or is this a known trade-off of the V1 architecture?

I would appreciate any insights into the DiscreteVAE training process. Thank you!
